packages/react/src/useInsertionEffect.ts

```diff
@@ -0,0 +1,7 @@
+import * as React from 'react';
+
+// @ts-expect-error
+export const useInsertionEffect = React['useInsertion' + 'Effect']
```

Comment:
Looks like this string concat is done to prevent issues with transpiling/bundling. Would be nice to have a comment explaining this, it wasn't obvious to me until I read [this emotion PR](https://github.com/emotion-js/emotion/pull/2651).